lib/media/drm_engine.js

```diff
@@ -1293,7 +1293,8 @@ shaka.media.DrmEngine = class {
       // However, unlike Edge and Chromecast, Tizen doesn't have this problem.
       if (this.currentDrmInfo_.keySystem == 'com.microsoft.playready' &&
           keyId.byteLength == 16 &&
-          !shaka.util.Platform.isTizen()) {
+          !shaka.util.Platform.isTizen() && 
+          !shaka.util.Platform.isVideoFutur()) {
```

Comment:
Do you think it might be better to say Platform.isEdge() instead of !Platform.isVideoFutur() ?

Reply:
I saw this suggestion by @fadomire in #2189 issue but I was wondering if putting `Platform.isEdge()` instead of `!shaka.util.Platform.isTizen() && !shaka.util.Platform.isVideoFutur()`will really cover all cases described in comments above and if it will not create some side effects (for Chromecast for example).

If you think that it will not introduce some other issues, I will make the change you suggest. But, in my opinion, it will be more logical to replace both conditions (`!shaka.util.Platform.isTizen() && !shaka.util.Platform.isVideoFutur()`) by `shaka.util.Platform.isEdge()`. What do you think about it?

Reply:
This will break WebOS. Please keep `!shaka.util.Platform.isTizen() && !shaka.util.Platform.isVideoFutur()`

Reply:
Okay.  Thanks, @avelad.

@LanaIV, it seems like you were right to take a conservative approach.